docker-compose.yml

```diff
@@ -225,3 +225,15 @@ services:
     entrypoint: []
     command: tail -F /dev/null
     user: node
+
+  pa11y:
+    container_name: "${PROJECT_NAME}_pa11y"
+    image: metadrop/docker-pa11y:latest
```

Comment:
Versions should be fixed for pa11y and lighthouse. Now they are using latest which makes the containers less stable as they could introduce an update that doesn't instantly works.

I propose:

- Add versions in .env and then use the version here, instead of latest
- Use 1.0 version for pa11y
- Use 0.13.0 version for lighthouse.

Reply:
We kept them as latest on purpose to have the latest version so the reports are always accurate, the integration is not as deep as other images like PHP or Selenium so newer versions will only affect the report calculation or HTML output.

Nevertheless, I fully open to discuss it :)

Reply:
I understand and i only have one question: are accesibility / performance rules updated so frequently?

Reply:
I agree in general we should control the releases but as @jorgetutor says, in this case the containers are totally external, no other containers or functionality depends on them, so at most only the functionality provided by them would break. Given that the functionality is not critical and I don't expect those containers to be shipped with a faulty release it seems more or less safe to use latest.

It would be a problem if they change how they are run (parameters, for example) or the configuration format, but I guess this is not very likely. And having to maintain the release number can be problematic as we need to check the available versions manually (the other containers provided by Wodby can be easily checked all at once on the last Wodby release).

So, I would leave "latest", but I'm open to discussion.

Reply:
I honestly don't know yet as both projects are quite new (for me), as example there was a release one month ago and the previous one was 9 months ago: https://hub.docker.com/r/patrickhulce/lhci-client/tags

I guess that new standards will be raised as the time comes